packages/table/preview/index.tsx

```diff
@@ -189,6 +189,19 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
             showRowHeadersLoading: false,
             showZebraStriping: false,
         };
+        this.state = {};
+        // populate the state from local storage, if the values are there, otherwise use (and store) the defaults
+        Object.keys(stateDefaults).forEach((stateKey: keyof IMutableTableState) => {
+            let stateValue = localStorage.getItem(stateKey);
```

Comment:
why not just have one root key for the entire state that contains `JSON.stringify(this.state)`?